docs/tutorials/index.md

```diff
@@ -7,48 +7,19 @@ type: markdown
-</p>
-
-<div id="toc" class="toc mobile-toc"></div>
-
-
-## Something Here
-
-### [Section 1]
-[Body Copy] Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit consequat finibus. Nullam ullamcorper ex sit amet laoreet mollis. Morbi quis porttitor sapein. Mauris malesuada libero a egestas iaculis.
-
- -  Internal site text link
- -  External link
-
-### [Section 2]
-[Body Copy] Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit consequat finibus. Nullam ullamcorper ex sit amet laoreet mollis. Morbi quis porttitor sapein. Mauris malesuada libero a egestas iaculis.
-
-```ruby
-// Create a default SSL ChannelCredentials object.
-auto channel_creds = grpc::SslCredentials(grpc::SslCredentialsOptions()); 
-// Create a channel using the credentials created in the previous step.
-auto channel = grpc::CreateChannel(server_name, creds);
-// Create a stub on the channel.
-std::unique_ptr<Greeter::Stub> stub(Greeter::NewStub(channel));
-// Make actual RPC calls on the stub.
-grpc::Status s = stub->sayHello(&context, *request, response);
-```
-
-#### [Section 2.1]
-[Body Copy]Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit consequat finibus. Nullam ullamcorper ex sit amet laoreet mollis. Morbi quis porttitor sapein. Mauris malesuada libero a egestas iaculis. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit consequat finibus. Nullam ullamcorper ex sit amet laoreet mollis. Morbi quis porttitor sapein. Mauris malesuada libero a egestas iaculis.
-
-#### [Section 2.2]
-[Body Copy]Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit consequat finibus. Nullam ullamcorper ex sit amet laoreet mollis. Morbi quis porttitor sapein. Mauris malesuada libero a egestas iaculis. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit consequat finibus. Nullam ullamcorper ex sit amet laoreet mollis. Morbi quis porttitor sapein. Mauris malesuada libero a egestas iaculis.
-
-
-### [Section 3]
-[Body Copy] Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit consequat finibus. Nullam ullamcorper ex sit amet laoreet mollis. Morbi quis porttitor sapein. Mauris malesuada libero a egestas iaculis.
-
-#### [Section 3.1]
-[Body Copy]Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit consequat finibus. Nullam ullamcorper ex sit amet laoreet mollis. Morbi quis porttitor sapein. Mauris malesuada libero a egestas iaculis. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit consequat finibus. Nullam ullamcorper ex sit amet laoreet mollis. Morbi quis porttitor sapein. Mauris malesuada libero a egestas iaculis.
+This section contains tutorials for each of our supported languages. They introduce you to gRPC's API and associated concepts, and the different RPC types that are available. If you just want to dive straight in with a working example first, see our [Quickstarts](../quickstart).
+
```

Comment:
Just realized it's probably better to do a more qualified link, ie [Quickstarts]({{site.baseurl}}/docs/quickstart)